packages/eslint-config-jest/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * strvcom/eslint-config-jest
+ *
+ * @author      Michal Jarnot <m.jarnot@yahoo.com>
+ * @copyright   2019 STRV
+ * @license     http://choosealicense.com/licenses/bsd-3-clause  BSD-3-Clause License
+ */
+
+'use strict'
+
+module.exports = {
+  extends: ['plugin:jest-formatting/recommended'],
+  plugins: ['jest', 'jest-formatting'],
+  overrides: [{
+    files: ['**/__tests__/**/*.[jt]s?(x)', '**/?(*.)+(spec|test).[jt]s?(x)'],
+    env: {
+      'jest/globals': true,
+    },
+  }],
+  rules: {
+    //
+    // Recommended
+    //
+
+    'jest/expect-expect': 'error',
+    'jest/no-disabled-tests': 'error',
+    'jest/no-focused-tests': 'error',
+    'jest/no-identical-title': 'error',
+    'jest/no-jasmine-globals': 'error',
+    'jest/no-jest-import': 'error',
+    'jest/no-mocks-import': 0,
+    'jest/no-test-callback': 'error',
+    'jest/no-test-prefixes': 'error',
+    'jest/valid-describe': 'error',
+    'jest/valid-expect-in-promise': 'error',
+    'jest/valid-expect': 'error',
+
+    //
+    // Optional
+    //
+
+    'jest/consistent-test-it': 'error',
+    'jest/lowercase-name': 'error',
+    'jest/no-hooks': 0,
+    'jest/no-large-snapshots': 'error',
+    'jest/no-test-return-statement': 'error',
+    'jest/no-truthy-falsy': 0,
+    'jest/prefer-called-with': 1,
```

Comment:
```suggestion
    'jest/prefer-called-with': 'error',
```
There is no reason to allow this, if we need to check whether a function was called we can use `toHaveBeenCalledTimes`.

Reply:
But we may have to check arguments it was called with? 🤔

Reply:
✅